editors/intellij/README.md

```diff
@@ -1,44 +1,55 @@
-# Biome JetBrains IDEs Plugin
 
-https://biomejs.dev plugin for JetBrains IDEs.
+[Biome](https://biomejs.dev/) is a powerful tool designed to enhance your development experience. This plugin integrates seamlessly with many [JetBrains IDE's](#Supported IDEs) to provide some capabilities:
```

Comment:
This is great! the updated readme is really helpful for users!

I think this should be reflected to META-INF/plugin.xml by follow up PR. (We've passed on using `org.jetbrains.changelog.markdownToHTML` in the past, but maybe it's time to use)